### ISSUE TITLE
Mismatched exe and bin naming

### ISSUE THREAD

**Issue by Earnestly** [dscharrer/innoextract#59]
I have an old game I made isos from long ago called "Lionheart: Legacy of the Crusade".  Lately I've been trying to automate various aspects of my small collection and found this tool actually worked with the setup system this game uses.

While I can work around this issue (explained at the end) I did think it might be worth reporting it.

The game normally runs `LanguageSelect.exe` (via the `Autorun.inf`) where the installer name is, for example, `English.exe`.  There are two bin files called `SetupLionheart-1.bin` and `SetupLionheart-2.bin`.

If I attempt to use `innoextract -t English.exe` I get the following error:

```
% innoextract -t English.exe
Testing "Lionheart: Legacy of the Crusader (TM)" - setup data version 3.0.6.1
Opening "English-1.bin"
Stream error while extracting files!
 └─ error reason: could not open slice 0: "English-1.bin": iostream error
If you are sure the setup file is not corrupted, consider 
filing a bug report at http://innoextract.constexpr.org/issues
Done with 1 error.

% innoextract -v English.exe
innoextract 1.6
Extracts installers created by Inno Setup 1.2.10 to 5.5.8
```

The work around is to rename `English.exe` to `SetupLionheart.exe` and innoextract works beautifully.

Lastly, thank you very much for this tool, regardless of what comes of this issue.  It has made my life a lot easier when it comes to automating installs.

**Comment by dscharrer** [comment]
This should be fixed in innoextract 1.7 (to be released soon), a [release candidate](http://constexpr.org/innoextract/files/snapshots/innoextract-1.7-rc1/) is available for testing. Both .bin names should work now. Please let me know if it still doesn't work for you.

**Comment by Earnestly** [comment]
Works perfectly here, thank you
